scripts/weight-diff.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# This is a helper script for generating comparison tables between branches
+
+set -e
+
+help() {
+  cat <<'EOF'
+Generate comparison tables with weights between branches
+
+USAGE:
+    ./weight-diff.sh <BRANCH1> <BRANCH2> <RUNTIME> <KIND>
+
+ARGUMENTS:
+  <BRANCH1>  branch #1 from where to get the weights
+  <BRANCH2>  branch #2 from where to get the weights
+  <RUNTIME>  what runtime to compare? [possible values: gear, vara]
+  <KIND>     for which weights to generate a table? [possible values: instruction, host-fn]
```

Comment:
Please add kind "all" and defaults:
branch 1 is current
branch 2 is master
runtime gear
kind all

Reply:
@breathx kind "all" is not supported but I can add it